src/frequenz/client/dispatch/__init__.py

```diff
@@ -4,5 +4,6 @@
 """Dispatch API client for Python."""
 
 from ._client import Client
+from .types import Dispatch
 
-__all__ = ["Client"]
+__all__ = ["Client", "Dispatch"]
```

Comment:
Same as with `test/__init__.py` here, I would make the `types` module private (`_types`), as it is very unlikely that anyone would like to use the `Dispatch` type without using the client.

Reply:
Hmm in this case, `Dispatch` is a very common type, but there are quite a few sub-types that are def. useful (Weekday Enum, Frequency) but not always required when just `Dispatch` is required

Reply:
Then just leave it in `types` and don't expose it here, symbols should be exposed only in one place. And also there is no benefit, with this, you are actually forcing the user to import `types`, even if they don't use **any** type from there, every time they do `import frequenz.client.dispatch`.

We have been switching between a more nested and a flatter structure for a while now, I'm not convinced which is best, but when it is very likely that you'll use many stuff and the 'submodules' are relatively small, I think the flatter structure makes things easier. I think that is the case for the client, so I would probably go with that, but if you prefer to keep `types` separate, OK, just don't mix both styles and make sure symbols are publicly exposed only in one place.